src/main/java/gov/usgs/owi/nldi/controllers/CharacteristicsController.java

```diff
@@ -75,4 +75,18 @@ public void getCharacteristicData(HttpServletRequest request, HttpServletRespons
 		}
 		logService.logRequestComplete(logId, response.getStatus());
 	}
+
+	@GetMapping(value="{featureSource}/{featureID}/basin")
+	public void getBasin(HttpServletRequest request, HttpServletResponse response,
+			@PathVariable(LookupDao.FEATURE_SOURCE) String featureSource,
+			@PathVariable(Parameters.FEATURE_ID) String featureID) throws IOException {
+		BigInteger logId = logService.logRequest(request);
+		try {
+			streamBasin(response, getComid(featureSource, featureID));
+		} catch (Throwable e) {
```

Comment:
I agree that if the code catches `Throwable`, it should be at a high level. However, I'm still unclear on the benefits of catching `Throwable` in the first place. If the code catches `Throwable`, then, it is catching `Error`s. In the eyes of the [Java maintainers](https://docs.oracle.com/javase/8/docs/api/java/lang/Error.html), a `reasonable application` should not try to catch `Error`s. Do you recall concrete cases in WQP where catching `Error`s in addition to `Exception`s was beneficial? Do you recall any cases in WQP where catching `Error`s was problematic?